src/card.tsx

```diff
@@ -145,15 +180,23 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                             buttonLink={buttonLink}
                         />
                     </div>
-                </section>
+                </MaybeSection>
             </Boxed>
         );
+        return onClose ? (
+            <Dismissable onClose={onClose} aria-label={ariaLabel}>
+                {content}
+            </Dismissable>
+        ) : (
+            content
+        );
```

Comment:
perhaps the Dismissable component could perform this logic inside (in fact it would be a MaybeDismissable), but you don't need to repeat this logic in other places